pkg/alertmanager/validation/v1alpha1/validation.go

```diff
@@ -312,6 +316,20 @@ func validateTelegramConfigs(configs []monitoringv1alpha1.TelegramConfig) error
 	return nil
 }
 
+func validateMSTeamsConfigs(configs []monitoringv1alpha1.MSTeamsConfig) error {
+	for _, config := range configs {
+		if config.WebhookURL == "" {
```

Comment:
By using the kubebuilder attribute as mentioned [here](https://github.com/prometheus-operator/prometheus-operator/pull/5778/files#r1284413898), we can remove this.